control/src/main/java/org/openbase/bco/dal/control/layer/unit/AbstractUnitController.java

```diff
@@ -159,6 +168,34 @@
     private final Map<ServiceType, Timeout> requestedStateCacheTimeouts;
     final BuilderSyncSetup<DB> builderSetup;
 
+    private static final String INFLUXDB_BUCKET_DEFAULT = "bco-persistence";
```

Comment:
Please create a JPInfluxDbBucketProperty instead declaring this constant multible times and use it in here, in the InfulxDBHandler class as well as in BCozy as default when not overwritten via the property file (globally search for ``bco-persistence``).